### PR TITLE
Added components for Inputs and Responsive images.

### PR DESCRIPTION
**Pull request by Anurag021**
Added files 
1. input.css,
2.  images.css and
3. added images and input components in docs.html.
4. also done some refactoring on color part of the code.
image
image
image
image

### CODE REVIEW

File: Components/avatar.css [hunk 1]
```diff
@@ -3,36 +3,30 @@
...
-
 .avatar{
     padding: 1rem;
     align-self: center;
 }
-
 .avatar-img{
     border-radius: 50%;
     height:100%;
     width:100%;
```
Comment (partha8) [inline Components/avatar.css:16]: add height: auto and have object-fit: cover here

**Comment by anik31** [comment]
You can give some height & width to the round image to make it a circle

File: Components/badges.css [hunk 2]
```diff
@@ -2,41 +2,33 @@
...
     padding: 1rem; 
 }
-
 .badge{
-    color:var(--color-white);
-    background: var(--color-brown);
+    color:var(--color-gray-100);
+    background: var(--color-primary);
     border-radius: 50%;
     padding: 0.1rem 0.5rem;
```
Comment (partha8) [inline Components/badges.css]: You can use padding: 2px 8px here, rem is not required as it wont make much difference

File: Components/input.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    margin:0.3rem;
+}
+#password{
+    border: 2px var(--color-error) solid;
+}
+.input-form input,#description {
+    width: 80%;
+}
+#form-submit{
+    width: 40%;
```
Comment (partha8) [inline Components/input.css]: avoid using id selectors in css https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4, refer this article

**Comment by partha8** [comment]
Like @anik31 mentioned, give height and width to make the image round.

File: Components/avatar.css (hunk 1 above)
Comment (Anurag021) [reply Components/avatar.css:16]: Object-fit: cover -- done,
height: auto, is affecting the avatar with badges one, it used to get out, hence didn't change.

File: Components/badges.css (hunk 2 above)
Comment (Anurag021) [reply Components/badges.css]: Thanks, this one done too.

File: Components/input.css (hunk 3 above)
Comment (Anurag021) [reply Components/input.css]: great read..... done replacing id with class.

**Review by partha8 — APPROVED** [review]

File: Components/alert.css [hunk 4]
```diff
@@ -1,40 +1,37 @@
...
 .alert-box{
     padding: 1rem;
-    border: 1px solid var(--color-black);
+    border: 1px solid var(--color-gray-600);
     margin: 0.5rem;
     border-radius:1rem;
     width: 60%;
     height: auto;
     display: flex;
     justify-content: center;
```
Comment (ShekharDhangar) [inline Components/alert.css:15]: you can use flex properties as helper class

File: Components/button.css [hunk 5]
```diff
@@ -85,7 +86,7 @@
   align-items: center;
   justify-content: center;
```
Comment (ShekharDhangar) [inline Components/button.css:82]: same use flex properties as helper class. In that way you dont have to repeat the code

File: Components/cards.css [hunk 6]
```diff
@@ -29,26 +29,26 @@
     justify-content: space-evenly;
   }
   .horizontal-card-button{
-    background-color: var(--color-primary-dark);
-    color: var(--color-white);
+    background-color: var(--color-primary);
+    color: var(--color-gray-100);
     border-radius: 5px;
     cursor: pointer;
```
Comment (ShekharDhangar) [inline Components/cards.css]: use pointer as helper class

**Review by ShekharDhangar — APPROVED** [review]
Try to do suggested changes, rest is good

File: Components/alert.css (hunk 4 above)
Comment (Anurag021) [reply Components/alert.css:15]: Great thanks, done the required changes, created a helper class for flex properties.

File: Components/cards.css (hunk 6 above)
Comment (Anurag021) [reply Components/cards.css]: Done

**Pull request merged**